### PR TITLE
Increase MSI to 100% + Beautify code + Improve readme + Update dev dependencies

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: src/Cookie.php [hunk 1]
```diff
@@ -116,7 +109,8 @@ final class Cookie
      * @var string|null Asserts that a cookie must not be sent with cross-origin requests.
      * This provides some protection against cross-site request forgery attacks (CSRF).
      *
-     * @see https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html#samesite-cookie-attribute More information about sameSite.
+     * @see https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html#samesite-cookie-attribute
```
Comment (roxblnfk) [inline src/Cookie.php:112]:
```suggestion
     * @link https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html#samesite-cookie-attribute
```

File: src/Cookie.php [hunk 2]
```diff
@@ -563,7 +559,10 @@ public static function fromCookieString(string $string): self
     }
 
     /**
+     * @param string $attribute
      * @psalm-return non-empty-list<null|string>
+     *
+     * @return array
```
Comment (roxblnfk) [inline src/Cookie.php:565]:
```suggestion
     * @psalm-return non-empty-list<null|string>
```

**Review by roxblnfk — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -85,21 +92,20 @@ The code is statically analyzed with [Psalm](https://psalm.dev/). To run static
...
-### Follow updates
+## Follow updates
 
 [![Official website](https://img.shields.io/badge/Powered_by-Yii_Framework-green.svg?style=flat)](https://www.yiiframework.com/)
 [![Twitter](https://img.shields.io/badge/twitter-follow-1DA1F2?logo=twitter&logoColor=1DA1F2&labelColor=555555?style=flat)](https://twitter.com/yiiframework)
 [![Telegram](https://img.shields.io/badge/telegram-join-1DA1F2?style=flat&logo=telegram)](https://t.me/yii3en)
 [![Facebook](https://img.shields.io/badge/facebook-join-1DA1F2?style=flat&logo=facebook&logoColor=ffffff)](https://www.facebook.com/groups/yiitalk)
 [![Slack](https://img.shields.io/badge/slack-join-1DA1F2?style=flat&logo=slack)](https://yiiframework.com/go/slack)
-
-## License
```
Comment (rustamwin) [inline README.md:100]: Why was it removed?

Comment (vjik) [reply README.md:100]: Move up, not remove.

File: src/Cookie.php (hunk 2 above)
Comment (vjik) [reply src/Cookie.php:565]: phpdoc without @param and @return... why?

Comment (rustamwin) [reply src/Cookie.php:565]: because phpdocs and typehints are the same.

**Review by rustamwin — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**